pvlib/tests/test_modelchain.py

```diff
@@ -1904,7 +1904,7 @@ def test_complete_irradiance(sapm_dc_snl_ac_system, location):
 
     mc.complete_irradiance(i[['dhi', 'ghi']])
     assert_series_equal(mc.results.weather['dni'],
-                        pd.Series([49.756966, 62.153947],
+                        pd.Series([49.63565561689957, 62.10624908037814],
```

Comment:
Hmm, I guess this is an artifact of switching to `apparent_zenith`.  Maybe we shouldn't actually make that change, or if we do then it needs to be mentioned in the changelog.  Let's keep this as "to discuss" also.